### PR TITLE
[Messenger] Improve handler options description

### PR DESCRIPTION
**Pull request by makmaoui**

### CODE REVIEW

File: messenger.rst [hunk 1]
```diff
@@ -2016,11 +2016,16 @@ You can configure your handler by passing options to the attribute::
 
 Possible options to configure with the attribute are:
 
-* ``bus``
-* ``fromTransport``
-* ``handles``
-* ``method``
-* ``priority``
+============================  ====================================================================================================
+     Option                   Description
```
Comment (OskarStark) [inline messenger.rst]:
```suggestion
Option                        Description
```

**Comment by OskarStark** [comment]
@alamirault we have some false/positives here from DOCtor-RST, do you want to have a look? 😄

File: messenger.rst (hunk 1 above)
Comment (makmaoui) [reply messenger.rst]: Fixed, thanks

**Comment by javiereguiluz** [comment]
This is much better. Thanks Rachid!

**Pull request merged**